src/freezing/freezing_main.py

```diff
@@ -8,26 +8,23 @@
 from src.staging.validation import validate_data_with_schema
 from src.utils.defence import type_defence
 from src.utils.helpers import convert_formtype
+from src.freezing.freezing_compare import get_additions, get_amendments, output_freezing_files
 
 
 FreezingLogger = logging.getLogger(__name__)
 
 
 def run_freezing(
-    main_snapshot: pd.DataFrame,
-    # secondary_snapshot: pd.DataFrame,
+    snapshot_df: pd.DataFrame,
     config: dict,
-    # check_file_exists: Callable,
     write_csv: Callable,
     read_csv: Callable,
     run_id: int,
 ) -> pd.DataFrame:
     """Run the freezing module.
 
     Args:
-        main_snapshot (pd.DataFrame): The staged and vaildated snapshot data.
-        secondary_snapshot (pd.DataFrame): The staged and validated updated
-            snapshot data.
+        snapshot_df (pd.DataFrame): The staged and vaildated snapshot data.
         config (dict): The pipeline configuration
         check_file_exists (callable): Function to check if file exists. This will
```

Comment:
check_file_exists appears to be removed from the function, but left in the docstring.